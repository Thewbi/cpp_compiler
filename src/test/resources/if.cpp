int funcInt() {
    return 3;
}

char funcChar() {
    return 'c';
}

int main()
{
    //funcInt();

/**/
    //if (1 > 0) {
    //if (1 > 'c') {
    if (1 > funcInt()) {
    //if (1 > funcChar()) {
        return 1;
    }

    return 0;
}
