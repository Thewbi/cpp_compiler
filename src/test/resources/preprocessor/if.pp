if (prsvec_1.prso == oindex_1.valua || prsvec_1.prso == oindex_1.every) {
	goto L900;
}


(

prsvec_1
(

prsvec_1
.
(

prso
prsvec_1
.
(

prso
prsvec_1
.
==
(

oindex_1
prso
prsvec_1
.
==
(

oindex_1
.
prso
prsvec_1
.
==
(

oindex_1
.
prso
prsvec_1
.
==
(

valua
oindex_1
.
prso
prsvec_1
.
==
(

valua
oindex_1
.
prso
prsvec_1
.
==
(


valua
oindex_1
.
prso
prsvec_1
.
==
||