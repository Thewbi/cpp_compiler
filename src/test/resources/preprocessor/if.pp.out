if ( prsvec_1 . prso == oindex_1 . valua prsvec_1 . prso == oindex_1 . every ) { 
	 goto L900 ; 
} 
